src/config/keys.ts

```diff
@@ -54,7 +54,20 @@ export const buildManyItemMembershipsKey = (ids?: UUID[]) => [
 export const buildItemLoginKey = (id?: UUID) => [ITEMS_KEY, id, 'login'];
 export const TAGS_KEY = 'tags';
 export const ITEM_TAGS_KEY = 'itemTags';
+export const itemTagsKeys = {
```

Comment:
LGTM but should we have a better name for the factory?

Reply:
What name were you thinking of ?